akka-remote/src/test/scala/akka/remote/artery/LargeMessagesStreamSpec.scala

```diff
@@ -125,5 +124,6 @@ class LargeMessagesStreamSpec extends ArteryMultiNodeSpec("""
     }
   }
 
-  def awaitResolve(selection: ActorSelection): ActorRef = Await.result(selection.resolveOne(3.seconds), 3.seconds)
+  def awaitResolve(selection: ActorSelection): ActorRef =
+    awaitAssert(selection.resolveOne(3.seconds).futureValue, 3.seconds)
```

Comment:
I find it strange that it is lost. I thought the binding of remoting was synchronous with the ActorSystem creation.

Note that the failing PR validation was for release-2.5. I think aeron-udp is the default transport there.

Reply:
"Actor not found for" is a timeout. The test fails after 5 seconds, which is also strange since the timeout here is 3 seconds.

Reply:
I'm pretty sure I got the same outcome by not starting the other actor system then looking up the actor with the original logic.

Reply:
sure, but here it's known that the other AS is started first, and it's verified that the actor is up and running on that side

Reply:
It has only happened once, so maybe worth not changing the test?
I'm thinking that if this is a real problem we would have more such issues (or it could even be considered a bug if a started AS would not be able to receive messages immediately).

Reply:
Yeah, I think you are right, let's close this PR